Shared/Store/BaseDataStore.swift

```diff
@@ -430,27 +457,46 @@ extension BaseDataStore {
     }
 
     private func shutdown() {
-        self.loginsStorage?.close()
+        loginsStorage?.close()
     }
 
     private func initializeLoginsStorage() {
-        let filename = "logins.db"
-
-        let profileDirName = "profile.lockbox-profile)"
-
-        // Bug 1147262: First option is for device, second is for simulator.
-        var rootPath: String
-        let sharedContainerIdentifier = AppInfo.sharedContainerIdentifier
-        if let url = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: sharedContainerIdentifier) {
-            rootPath = url.path
+        guard let loginsDatabasePath = loginsDatabasePath else { return }
+        setupSalt()
```

Comment:
Nit: my preference would be to have `setupSalt()` return a string, and assignment happen here.

e.g.

```swift
self.salt = findSalt()
```